mkdocs.yml_template

```diff
@@ -64,115 +65,136 @@ nav:
-          - "On Heroku":
-            - "Deploying on Heroku": manuals/run-deploy/deploy/heroku/index.md
-            - "Setup": manuals/run-deploy/deploy/heroku/setup.md
-            - "Based on Git": manuals/run-deploy/deploy/heroku/git.md
-            - "Based on Docker": manuals/run-deploy/deploy/heroku/docker.md
-          - "On Databricks":
-            - "Deploying on Databricks": manuals/run-deploy/deploy/databricks/index.md
+            - manuals/userman/gui/extension/dynamic_element/index.md
+            - "Scalar properties": manuals/userman/gui/extension/dynamic_element/scalar_props.md
+
+      - "Data integration":
+        - "Data integration": manuals/userman/data-integration/index.md
+        - "Data node config": manuals/userman/data-integration/data-node-config.md
+        - "Data node usage": manuals/userman/data-integration/data-node-usage.md
+        - "Visual elements": manuals/userman/data-integration/data-node-vizelmts.md
+        - "Data node history and validity": manuals/userman/data-integration/data-node-history.md
+
+      - "Task orchestration":
+        - "Scenarios for task orchestration": manuals/userman/task-orchestration/index.md
+        - "Hello world!": manuals/userman/task-orchestration/hello-world.md
+        - "Scenario config": manuals/userman/task-orchestration/scenario-config.md
+        - "Scenario creation": manuals/userman/task-orchestration/scenario-creation.md
+        - "Scenario submission": manuals/userman/task-orchestration/scenario-submission.md
+        - "Visual elements": manuals/userman/task-orchestration/vizelmts.md
+
+      - "What-if analysis":
+        - "Scenarios for what-if analysis": manuals/userman/what-if-analysis/index.md
+        - "Multiple scenarios": manuals/userman/what-if-analysis/multiple-scenarios.md
+        - "Recurrent scenarios": manuals/userman/what-if-analysis/scenarios-and-cycles.md
+        - "Scenario comparison": manuals/userman/what-if-analysis/scenario-comparison.md
+
+      - "Scenario and data management":
+        - "Scenario and data management": manuals/userman/sdm/index.md
+        - "Data node":
+          - "Data node": manuals/userman/sdm/data-node/index.md
+        - "Task":
+          - "Task": manuals/userman/sdm/task/index.md
+        - "Scenario":
+          - "Scenario": manuals/userman/sdm/scenario/index.md
+          - "Scenario config": manuals/userman/sdm/scenario/scenario-config.md
+        - "Submission":
+          - "Submission": manuals/userman/sdm/submission/index.md
+        - "Job":
+          - "Job": manuals/userman/sdm/job/index.md
+        - "Sequence":
+          - "Sequence": manuals/userman/sdm/sequence/index.md
+        - "Cycle":
+          - "Cycle": manuals/userman/sdm/cycle/index.md
+
+      - "Configuration":
+        - "Configuration": manuals/userman/configuration/index.md
+        - "Configuring GUI service": manuals/userman/configuration/gui-config.md
```

Comment:
```suggestion
        - "Configuring the GUI service": manuals/userman/configuration/gui-config.md
```
There's more than the GUI service in there. So we must talk.

Reply:
@FabienLelaquais , Should we rename this to "Configuring the Gui"?